src/main/flight/mixer.c

```diff
@@ -66,7 +66,7 @@ PG_RESET_TEMPLATE(mixerConfig_t, mixerConfig,
     .yaw_motors_reversed = false,
 );
 
-PG_REGISTER_WITH_RESET_FN(motorConfig_t, motorConfig, PG_MOTOR_CONFIG, 0);
+PG_REGISTER_WITH_RESET_FN(motorConfig_t, motorConfig, PG_MOTOR_CONFIG, 1);
 
```

Comment:
That magic last paramter "1" should have a symbol name like `PG_MOTOR_CONFIG_VERSION`

Reply:
Something to look into when we do parameter group reorganisation - there are 50 or so of these values all over the code.

Reply:
What will it bring? If you want to document that it is version, use something like `#define PG_VERSION(x) x`

Reply:
Well, what ever makes it obvious that this is a version number for this struct. You have to look up the definition of  `PG_REGISTER_WITH_RESET_FN` etc etc. to find out what it is. But is this really a `PG_VERSION`and not a `PG_MOTOR_CONFIG_VERSION` ?? Two different things. 
Amazing we are arguing on such a trivial and basic thing. We enforce symbolic names over magic numbers in every other PR, why not here? All about clarity and human readability, without having to research the meaning of items.  If we are to change in 50 or so other places, why not start here and set a new standard. One place at a time, incremental evolution. No need for a big bang revolution.

Reply:
I'm not big fan of useless symbolic names. PG version is used only in one place. But you have a point about version location, it should be close to PG_DECLARE or structure definition.

It may be best to infer version in PG_REGISTER_WITH_RESET_FN, using `PG_MOTOR_CONFIG ## _VERSION`. Eventually even type and name may be inferred and specified only in one place ...

Reply:
@ledvinap:

> It may be best to infer version in PG_REGISTER_WITH_RESET_FN, using PG_MOTOR_CONFIG ## _VERSION. Eventually even type and name may be inferred and specified only in one place ...

That is a good suggestion - I have noticed in the past that it is suboptimal and an invitation to be overlooked that the version has to be set in a different file to the actual change of the struct.